modules/fedimint-dummy-tests/tests/tests.rs

```diff
@@ -70,3 +78,29 @@ async fn federation_should_abort_if_balance_sheet_is_negative() -> anyhow::Resul
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn unbalanced_transactions_get_rejected() -> anyhow::Result<()> {
```

Comment:
`/// Make sure that a tx with more output value than inputs get rejected`?

But why is this test in a dummy module in the first place?

Reply:
You need some module to generate an unbalanced tx, the dummy module is the most straight-forward. We could have some global integration testing crate again, but it wouldn't do anything differently imo, so we can just put it there. 

For the other test that checks the audit we actually need the dummy module because there we can patch in an accounting bug.

Reply:
Added a minimal commentary